contrib/buildahimage/README.md

```diff
@@ -22,6 +22,8 @@ The container images are:
   * quay.io/buildah/testing - This image is built using the latest version of Buildah that is or was in updates testing for Fedora.  At times this may be the same as the stable image.  This container image will primarily be used by the development teams for verification testing when a new package is created.  Built with buildahimage/testing/Dockerfile.
   * quay.io/buildah/stable:version - This image is built 'by hand' using a Fedora based container.  An RPM is first pulled from the [Fedora Updates System](https://bodhi.fedoraproject.org/) and the image is built from there.  For more details, see the Containerfile used to build it, buildahimage/stablebyhand/Containerfile.buildahstable
 
+`quay.io/buildah/upstream:master`, `quay.io/buildah/stable:version`, `quay.io/containers/buildah:version` images are multi-arch ones and available for `linux/amd64`, `linux/s390x`, `linux/arm64`, `linux/ppc64le` architectures.
+
```

Comment:
@barthy1 I've got to apologize for being a complete and total dunderhead.  The automated builds for Buildah should NOT be making `quay.io/buildah/stable:version` or `quay.io/containers/buildah:version`, or at the very least they should make them one and only one time.  The issue is some of our customers pull those images and as a security check, see if the manifest has changed since they last used/pulled it.  If it has, they flag the image as being corrupt.  

So we need to back out those two builds from the yaml or figure out a way to build them one and only one time. 

Also, we probably ought to change the `:master` tag to `:main`.

Again, apologies for not catching this sooner, and thanks for putting up with me.

(Tom goes a find a hole to crawl into so @barthy1 's bombs don't hurt too much...)

Reply:
@TomSweeneyRedHat 

> The issue is some of our customers pull those images and as a security check, see if the manifest has changed since they last used/pulled it. If it has, they flag the image as being corrupt.

Sorry, it seems I misunderstood the issue with skopeo where you pinged me. Now I understand it better. Nevertheless, at this moment version tag of the skopeo container image  doesn't fit actual version of the binary inside the image. Just reminder.

> figure out a way to build them one and only one time

hmm, interesting task. Let me work on it. If I don't find solution tomorrow with pushing `:v` images just one time, then I will just switch off this part of the build.. + change `:master` to `:main`. And it's great time for you to decide if any other tags should be multi-arched (for instance, `latest`?) It won't make the build longer, as actual build is done just one time.

> Again, apologies for not catching this sooner, and thanks for putting up with me.

Don't worry, I like doing multi-arch things for container community, and you just gave me one more task :)))